lib/rtoolsHCK.rb

```diff
@@ -345,61 +327,208 @@ def deploy_tools_hck(script_file)
     logger('debug', 'initialize/toolsHCK') { 'toolsHCK deployed on guest!' }
   end
 
+  # toolsHCK server TCP socket listener's port base
+  TOOLSHCK_TCP_SOCKET_PORT_BASE = 4000
```

Comment:
I suggest to get the ports from the configuration json file (maybe keep those as defaults).